tests/blackbox_test.go

```diff
@@ -42,6 +42,10 @@ var (
 	testTimeout = time.Second * 30
 )
 
+type myTest struct {
```

Comment:
Why define this type rather than adding the methods directly to `types.Test`?

Reply:
@ajeddeloh mentioned this above. I just fixed it, will push it now.